prometheus-to-sd/README.md

```diff
@@ -44,6 +44,14 @@ the prometheus-to-sd. Values of the `namespace_id` and `pod_id` can be passed to
 the component through the additional flags or omitted. `container_name` is
 always empty for now. Field `zone` is overridable via flag.
 
+From 0.8.0, Prometheus can also take `nodeLabelsOverride` flag to override all MonitoredResource labels as a k8s node.
```

Comment:
Please also add a note that when this mechanism is used, prom-to-sd will not automatically determine any labels from GCE metadata server, all of them will have to be explicitly provided.

Reply:
PTAL.